communication/ws.go

```diff
@@ -58,10 +58,13 @@ func wsEndpoint(w http.ResponseWriter, r *http.Request) {
 
 	log.Println(player.Name + " has connected")
 
+	updateRocketChat(lobby, player)
```

Comment:
I think it'd be good to run this in a separate go routine, as it's not gameplay-critical and there's no need to increase latency over it. Same for the one in the close-handler.